sensing/geo_pos_conv/README.md

```diff
@@ -0,0 +1,21 @@
+# geo_pos_conv
+
+## Purpose
+
+The `geo_pos_conv` is a library to calculate the conversion between **x, y positions on the plane rectangular coordinate** and **latitude/longitude on the earth**.
+
+## Inner-workings / Algorithms
+
+## Inputs / Outputs
```

Comment:
@1222-takeshi 
can you briefly write this?

Reply:
Since this package is a library, it has no input or output.
As you can see from the Purpose, it is a simple transformation, so there is no particular algorithm.
Should I remove the Inputs / Outputs etc.?

Reply:
I see then you don't have to write I/O.